WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -18,9 +18,10 @@ Variables:
      associated_events.find_by_event_id(event.id) || associated_events.build(event_id: event.id)
    end %>
 
-<div class="form-group">
-  <%= label_tag events_association_name, label, class: "associated-events-label" %>
-  <div id="<%= events_association_name %>" class="associated-events">
+<% errors = form_builder.object.errors.messages[events_association_name] || [] %>
+<div class="form-group <%= "has-error" if errors.length > 0 %>">
+  <%= label_tag events_association_name, label, class: "control-label col-sm-2" %>
+  <div id="<%= events_association_name %>" class="col-sm-9">
```

Comment:
Ah, good catch! I removed these classes while debugging something else, and then was too lazy to add them back. I didn't realize I changed the behavior.